views/prisonerProfile/prisonerWorkAndSkills/prisonerWorkAndSkills.njk

```diff
@@ -6,7 +6,7 @@
 
 {% set title = "Work and Skills" %}
 {% set containerClasses = "prisoner-profile" %}
-{% set missingLevel = "Missing data" %}
+{% set apiErrorMessage = "Unable to show this detail." %}
```

Comment:
**Non blocking**
I wonder if a view need to know about an api?
The data wend from api -> service -> routeHandler -> browser.

Perhaps the view should stay decoupled from the concept of `api`. 

Maybe something like `fallbackMessage` or `errorMessage`?